docker/build.sh

```diff
@@ -2,14 +2,16 @@
 
 SCRIPT_DIR=$(dirname "$0")
 ROOT_DIR=$(git rev-parse --show-toplevel)
+TAG=$(git rev-parse --short=6 HEAD)
+ORG=${1:-interactem}
 
-podman build -t interactem/interactem -f $SCRIPT_DIR/Containerfile.base $ROOT_DIR/backend
+docker build --platform=linux/amd64,linux/arm64 -t interactem/interactem:$TAG -t interactem/interactem:latest -f $SCRIPT_DIR/Containerfile.base $ROOT_DIR/backend
 if [ $? -ne 0 ]; then
     echo "Failed to build interactem/interactem"
     exit 1
 fi
 
-podman build -t interactem/operator -f $SCRIPT_DIR/Containerfile.operator $ROOT_DIR/backend
+docker build --platform=linux/amd64,linux/arm64 -t $ORG/operator:$TAG -f $SCRIPT_DIR/Containerfile.operator $ROOT_DIR/backend
```

Comment:
Don't we still want to be able to build with podman?

Reply:
Yes however the multiplatform build stuff in podman doesn't automatically use a manifest, which apparently is required... : https://blog.while-true-do.io/podman-multi-arch-images/

So it is not as simple as the docker interface. I was thinking we could keep this as docker, push to a repo, and pull down when building individual operators. 

But we could specify an option (`-p`?) that would default to building without multiplatform support. Eventually we should have all of this done in CI anyway...